packages/kit/src/core/adapt/prerender.js

```diff
@@ -90,6 +90,19 @@ export async function prerender({ cwd, out, log, config, build_data, fallback, a
 		}
 	});
 
+	/**
+	 * @param {string} path
+	 */
+	function normalizeTrailingSlash(path) {
+		if (config.kit.trailingSlash === 'always') {
+			return path.endsWith('/') || path === '/' ? path : `${path}/`;
```

Comment:
you don't need `|| path === '/'` do you? I think that `path.endsWith('/')` would return `true` for it

Reply:
True 🙈